src/mediaplayer.cs

```diff
@@ -1409,7 +1418,7 @@ interface MPNowPlayingInfoCenter {
 		NSString PropertyIsLiveStream { get; }
 
 		[iOS (10,3)]
-		[Mac (10,12,3)]
+		[TV (10,3)]
```

Comment:
The docs said TVOS 10,3 :(
https://developer.apple.com/documentation/mediaplayer/mpnowplayinginfopropertyasseturl

The headers dont specify: MP_EXTERN NSString *const MPNowPlayingInfoPropertyAssetURL NS_AVAILABLE(10_12_3, 10_3); // NSURL